app.js

```diff
@@ -0,0 +1,77 @@
+
+console.log("Hello, World!"); //->English
+console.log("Silav, Cihan!");//->Kurdish
+console.log("Hoi, Wereld!"); //->Dutch
+
+console.log("I'm awesome!");
+
+let x =

```

Comment:
I don't think this will compile:) 

I think you wanted to do this?
`let x;` ?

Reply:
let x;

Reply:
It is solved